aioftp/server.py

```diff
@@ -972,7 +973,7 @@ def __init__(self,
         connection = Connection(
             client_host=host,
             client_port=port,
-            server_host=self.server_host,
+            server_host=server_host or self.server_host,
```

Comment:
Your answer does not correlate with my question. I'm talking about order of host resolution.
``` python
server=host=self.server_host or writer.transport.get_extra_info("sockname")[0]
```
So, if `server_host` exist, then it should be used.